app/dashboard/models.py

```diff
@@ -1633,6 +1633,18 @@ def to_dict(self, fields=None, exclude=None):
         return model_to_dict(self, **kwargs)
 
 
+class LabsResearch(models.Model):
+    """Define the structure of Labs Research object."""
+
+    title = models.CharField(max_length=255)
+    description = models.CharField(max_length=1000)
+    link = models.URLField(null=True)
+    img_url = models.URLField(null=True)
```

Comment:
This is a no-go.  We either need to restrict to the `https` scheme or better yet use an `ImageField` here and allow uploading said image asset, so we know it's *always* available and served over HTTPS... otherwise we could encounter serving mixed content.

Reply:
Sure, I'll change it to ImageField